### DISCUSSION TITLE
composer install giving error on autoload files

### DISCUSSION THREAD

**Discussion by batoolfatima2135** [mettle/sendportal#203] [Help]
hey,
i have cloned this repo but every time i run composer install it give this error i m unable to solve:

Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi
PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in C:\xampp\htdocs\sendportal\vendor\psr\log\src\LoggerInterface.php on line 30
PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in C:\xampp\htdocs\sendportal\vendor\psr\log\src\LoggerInterface.php on line 30
PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255

kindly guide me what im doing wrong, im using php 7.4 composer 2.2.6 

here is screenshot of my error:
![Screenshot (23)](https://user-images.githubusercontent.com/61277612/152633474-f934d609-ffb6-4d56-898d-9ebd9bfd9a40.png)

**Comment by Reayan-Iqbal** [comment]
@batoolfatima2135  Did you find any solution?
